shared/src/main/java/com/faforever/neroxis/map/SCMap.java

```diff
@@ -120,7 +120,9 @@ public SCMap(int size, Biome biome) {
         navalRallyMarkers = new ArrayList<>();
         waveGenerators = new ArrayList<>();
         cubeMaps = new ArrayList<>();
-        cubeMaps.add(new CubeMap("<default>", "/textures/environment/defaultenvcube.dds"));
+        for (CubeMap cubeMap : biome.getTerrainMaterials().getCubeMaps()) {
+            addCubeMap(cubeMap);
+        }
```

Comment:
Since the biome contains the cubeMaps we can remove the addCubeMap method and just use the ones embedded in the biome for import and export